superdesk/io/commands/remove_expired_content.py

```diff
@@ -36,25 +33,31 @@ def run(self, provider_name=None):
                 self.remove_expired(provider)
 
     def remove_expired(self, provider):
+        lock_name = 'ingest:gc'
+
+        if not lock(lock_name, expire=300):
```

Comment:
My bad, I thought it was `if host is None` then you can have two task running on two host (in HA). Negative side is more concurrency issues. 
We are running multiple celery beat so it creates multiple task and locks are host-aware as we are either passing blank or none. In this case, we are seeing two task trying to process same item twice. 
We are going to pass a dummy value for host so that there is only one lock across host. see ticket SD-4827.